packages/design-system/src/components/N8nTags/Tags.vue

```diff
@@ -1,44 +1,87 @@
 <template>
 	<div :class="['n8n-tags', $style.tags]">
-		<n8n-tag v-for="tag in tags" :key="tag.id" :text="tag.name" @click="$emit('click', tag.id, $event)"/>
+		<n8n-tag v-for="tag in visibleTags" :key="tag.id" :text="tag.name" @click="$emit('click', tag.id, $event)"/>
+		<n8n-link
+			v-if="truncate && !showAll && hiddenTagsLength > 0"
+			theme="text"
+			underline
+			size="small"
+			@click.stop.prevent="showAll = true"
+		>
+			+{{ hiddenTagsLength }} more
```

Comment:
i18n?

Reply:
Good point. Also added story for truncated tags.